cli/file_fetcher.rs

```diff
@@ -220,7 +221,7 @@ impl FileFetcher {
     http_cache: HttpCache,
     cache_setting: CacheSetting,
     allow_remote: bool,
-    ca_data: Option<Vec<u8>>,
+    root_store: Option<RootCertStore>,
```

Comment:
`root_store` is quite ambiguous, how about using `root_cert_store` for variables that are of this type?

Reply:
no problem